x-pack/plugins/security_solution/public/management/components/endpoint_response_actions_list/response_actions_log.test.tsx

```diff
@@ -382,42 +394,6 @@ describe('Response Actions Log', () => {
     });
   });
 
-  describe('With agentIds filter', () => {
-    it('should NOT show a host column when a single agentId', async () => {
-      const agentIds = uuid.v4();
-      mockUseGetEndpointActionList = {
-        ...baseMockedActionList,
-        data: await getActionListMock({ actionCount: 2, agentIds: [agentIds] }),
-      };
-      render({ agentIds });
-
-      expect(
-        Array.from(
-          renderResult.getByTestId(`${testPrefix}-table-view`).querySelectorAll('thead th')
-        )
-          .slice(0, 5)
-          .map((col) => col.textContent)
-      ).toEqual(['Time', 'Command', 'User', 'Comments', 'Status']);
-    });
-
-    it('should show a host column when multiple agentIds', async () => {
-      const agentIds = [uuid.v4(), uuid.v4()];
-      mockUseGetEndpointActionList = {
-        ...baseMockedActionList,
-        data: await getActionListMock({ actionCount: 2, agentIds }),
-      };
-      render({ agentIds });
-
-      expect(
-        Array.from(
-          renderResult.getByTestId(`${testPrefix}-table-view`).querySelectorAll('thead th')
-        )
-          .slice(0, 6)
-          .map((col) => col.textContent)
-      ).toEqual(['Time', 'Command', 'User', 'Host', 'Comments', 'Status']);
-    });
-  });
-
```

Comment:
The `Host` column is now driven by the `showHostsInfo` flag, so no need for these tests. I've updated tests to account for the flag value [here](https://github.com/elastic/kibana/pull/139379/commits/9d6759adac93091e16b2db888f7b29c3ab82dc2f#diff-05b661516ad4879a260c5a51440309dad4a64c5dd1231e8c463e6af8214079f1L189-R201)